tests/test_decomposition_fiches_de_paie.py

```diff
@@ -6,7 +6,8 @@
 import os
 
 from openfisca_core import decompositions
-from cache import tax_benefit_system
+from openfisca_core.commons import to_unicode
+from .cache import tax_benefit_system
```

Comment:
Absolute imports :)

Reply:
The `test` package is not under `openfisca_france`, so I don't see how I can use an absolute import.